src/models/category.js

```
@@ -0,0 +1,38 @@
+    unique: [true, DOCUMENT_ALREADY_EXISTS('name')],
+    minLength: [1, FIELD_CANNOT_BE_SHORTER('name', 1)],
+    maxLength: [150, FIELD_CANNOT_BE_LONGER('name', 150)]
+  },
+  appearance: {
+    icon: {
+      type: String,
+      required: [true, FIELD_CANNOT_BE_EMPTY('icon')],
+      default: DEFAULT_CATEGORY_ICON
+    },
+    color: {
+      type: String,
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)